tests/test_helper.py

```diff
@@ -1,11 +1,23 @@
 import namanager.tests.helper as helper
 import sys
 import itertools
+import json
+
+# Backward Compatibility
+try:
```

Comment:
Any way you could break up this file into it's own block? (Maybe in a file?)

Reply:
I want and tried, but I couldn't find a way. I'll add TODO here.